tests/test_acm/test_acm.py

```diff
@@ -860,3 +861,24 @@ def test_elbv2_acm_in_use_by():
         _cert for _cert in certificates if _cert["CertificateArn"] == certificate_arn
     ][0]
     assert cert["InUse"]
+
+
+@mock_aws
+def test_certificate_expiration_status():
+    client = boto3.client("acm", region_name="eu-central-1")
+    arn = _import_cert(client)
+
+    resp = client.describe_certificate(CertificateArn=arn)
+    assert resp["Certificate"]["Status"] == "ISSUED"
+
+    expiry_time = resp["Certificate"]["NotAfter"]
+
+    # Test before expiry (1 second before)
+    with freeze_time(expiry_time - datetime.timedelta(seconds=1)):
```

Comment:
Freezing time is not possible in ServerMode, unfortunately, that is (presumably) why the tests are failing. 

We tend to just skip the tests if that's the case, see other ACM tests for an example:

https://github.com/getmoto/moto/blob/701c6e5f77b93cc3772f5a892500c62084d6705d/tests/test_acm/test_acm.py#L478:L479

Reply:
Thanks @bblommers, the issue is now fixed!